### PR TITLE
Recent opened files handling + other tweaks

### PR DESCRIPTION
**Pull request by FisheyLP**

### CODE REVIEW

File: src/us/deathmarine/luyten/MainMenuBar.java [hunk 1]
```diff
@@ -507,7 +568,7 @@ public void actionPerformed(ActionEvent event) {
 				pane.add(new JLabel("Contributions By:"));
 				pane.add(new JLabel("zerdei, toonetown, dstmath"));
 				pane.add(new JLabel("virustotalop, xtrafrancyz"));
-				pane.add(new JLabel("mbax, quitten, and mstrobel"));
+				pane.add(new JLabel("mbax, quitten, mstrobel, and FisheyLP"));
```
Comment (deathmarine) [inline src/us/deathmarine/luyten/MainMenuBar.java]: Nice.  :)

File: src/us/deathmarine/luyten/MainWindow.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
 public class MainWindow extends JFrame {
 	private static final long serialVersionUID = 5265556630724988013L;
 
-	private static final String TITLE = "Luyten";
+	private static final String TITLE = "Luyten 2";
```
Comment (deathmarine) [inline src/us/deathmarine/luyten/MainWindow.java]: This can be reverted.

Comment (FisheyLP) [inline src/us/deathmarine/luyten/MainWindow.java]: This wasn't intended to be in the pull request.. It was just for testing purposes.

**Comment by deathmarine** [comment]
Been a little busy but I just pulled your changes, everything looks great. Thanks.

**Pull request merged**